Maestro/subscriptions.json

```diff
@@ -630,6 +630,35 @@
           ]
         }
       }
+    },
+    // Update dependencies in WCF release/uwp6.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs/release/uwp6.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/uwp6.0/Latest.txt",
```

Comment:
I just removed it, we can always add it back in when it's ready.

I don't have permissions to merge, so please do when you thing it is good to go.